### PR TITLE
Rename isc -> iscmove

### PR DESCRIPTION
**Pull request by dessaya**

### CODE REVIEW

File: sui-go/sui/api_impl.go [hunk 1]
```diff
@@ -60,6 +60,6 @@ func NewSuiWebsocketClient(url string) *ImplSuiAPI {
 }
 
 const (
-	DefaultGasBudget uint64 = 10000000
+	DefaultGasBudget uint64 = 100000000
```
Comment (howjmay) [inline sui-go/sui/api_impl.go]: I think we should change the gas budget in iscmove only

File: sui-go/iscmove/client.go [hunk 2]
```diff
@@ -14,18 +14,19 @@ import (
...
 )
 
+// Client provides convenient methods to interact with the `isc` Move contracts.
 type Client struct {
-	// API *sui.ImplSuiAPI
 	*sui.ImplSuiAPI
 }
 
-func NewIscClient(api *sui.ImplSuiAPI) *Client {
+func NewClient(api *sui.ImplSuiAPI) *Client {
```
Comment (howjmay) [inline sui-go/iscmove/client.go:22]: I am curious about if we rename it to `NewClient()` isn't confusing with the sui client one?

Comment (dessaya) [reply sui-go/iscmove/client.go:22]: It's one of the suggestions here: https://go.dev/doc/effective_go#package-names

It's about avoiding repetition. The package name already contains `isc` so saying `isc.NewIscClient()` is repetitive. `isc.NewClient()` is more idiomatic.

File: sui-go/sui/api_impl.go (hunk 1 above)
Comment (dessaya) [reply sui-go/sui/api_impl.go]: Ok, I'll revert this change for now.

**Pull request merged**